repos/system_upgrade/el7toel8/actors/storagescanner/libraries/storagescanner.py

```diff
@@ -75,65 +75,75 @@ def _get_partitions_info(partitions_path):
                     name=name)
 
 
-@aslist
 def _get_fstab_info(fstab_path):
-    """ Collect storage info from /etc/fstab file """
-    if _is_file_readable(fstab_path):
-        with open(fstab_path, 'r') as fstab:
-            for line, entry in enumerate(fstab, 1):
-                if entry.startswith('#'):
-                    continue
-
-                entry = entry.strip()
-                if not entry:
-                    continue
-
-                entries = entry.split()
-
-                if len(entries) == 4:
-                    entries.append('0')
-
-                if len(entries) == 5:
-                    entries.append('0')
-
-                if len(entries) != 6:
-                    if any(value.startswith('#') for value in entries):
-                        remediation = (
-                            'Comments in the /etc/fstab file must be at the beginning of the line, your file has a'
-                            ' comment at the end of the line at line {}, please edit and fix this, for further'
-                            ' information read fstab man page (man 5 fstab).'.format(line)
-                        )
-                    else:
-                        remediation = (
-                            'The /etc/fstab file must have at least 4 values and at most 6 per line, your file on the'
-                            ' line: {} have {} values, please edit and fix this, for further information read'
-                            ' fstab man page (man 5 fstab). '.format(line, len(entries))
-                        )
-                    summary = (
-                        'The fstab configuration file seems to be invalid. You need to fix it to be able to proceed'
-                        ' with the upgrade process.'
+    """ Collect storage info from /etc/fstab file
+        Returns a list of successfully processed fstab entries, errors_dict
+    """
```

Comment:
```suggestion
    """
    Collect storage info from /etc/fstab file
    
    Return a list of successfully processed fstab entries and errors_dict
    """
```
Actually it returns tuple of two lists.